docs/guide/nested-fields.rst

```diff
@@ -283,6 +283,61 @@ populator of the relationship.
 If you don't have an existing type for creating a user, e.g. the "CreateCatInput" we used above,
 you can set the type to "auto", which will create a new type.
 
+Many to many with `through` models
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Using `graphene-django-cud`, a `through` model is more akin to a many to one relationship, as it is
+an intermediate model that connects two other models with itself.
```

Comment:
I think we should rewrite this to specify that this is how **Django** handles `through` fields internally (via a ManyToOneRel, although we don't need to specify that). This leads to many_to_one_extras being a more viable option to use with `through` intermediary models.  

It is not something we enforce, it is something Django itself enforces.